perlin.py

```diff
@@ -0,0 +1,151 @@
+import os
+import sys
+import xml.etree.ElementTree as ET
+from pprint import pprint
+from xml.etree import ElementTree
+import random
+
+from render import toimage
+
+import noise
+import numpy as np
+
+if 'SUMO_HOME' in os.environ:
+    tools = os.path.join(os.environ['SUMO_HOME'], 'tools')
+    sys.path.append(tools)
+else:
+    sys.exit("Please declare environment variable 'SUMO_HOME' to use sumolib")
+
+import sumolib
+
+# The 'noise' lib has good resolution until above 10 mil, but a SIGSEGV is had on values above [-100000, 100000]
+# FIXME: these values result in a noisemap which contains CRT-like lines and patterns. Find better, sane values.
+# POPULATION_BASE = random.randrange(-1000, 1000)
+POPULATION_BASE: float = 0
+# INDUSTRY_BASE = random.randrange(-1000, 1000)
+INDUSTRY_BASE: float = 0
+
+
+def get_edge_pair_centroid(coords: list) -> (float, float):
```

Comment:
`list` is a function. The type hint for lists are `List[T]` and for tuples are `Tuple[T, U, ..]`. so it should be:
```python
def get_edge_pair_centroid(coords: List[Tuple[float, float]]) -> Tuple[float, float]:
    ...
```
This is probably relevant other places as well, but I don't want to be super strict about it. Just wanted to let you know

Reply:
Cool, I tried fiddling around with it but didn't realise that `list != List`.
Fixed in 1ceff2d, it seems that's the only place that type is passed.